Corona-Warn-App/config/detekt.yml

```diff
@@ -100,7 +100,7 @@ complexity:
     ignoreStringsRegex: '$^'
   TooManyFunctions:
     active: true
-    excludes: ['**/test/**', '**/androidTest/**', '**/*.Test.kt', '**/*.Spec.kt', '**/*.Spek.kt', '**/LocalData.kt', '**/formatter/*Helper.kt']
+    excludes: ['**/test/**', '**/androidTest/**', '**/*.Test.kt', '**/*.Spec.kt', '**/*.Spek.kt', '**/LocalData.kt', '**/formatter/*Helper.kt', '**/viewmodel/*']
```

Comment:
Why is this necessary?

Reply:
@AlexanderAlferov

Reply:
because otherwise we could have only 20 methods max

Reply:
Correct, this is for readability and better understanding of code. Please try to extract code out of the class if you need more than 20 methods

Reply:
I agree that incase of ViewModels it would make code less readable to abstract it just due to method count. Let's go forward with the exclusion but please use `**/*ViewModel.kt` (test this before).